src/libstrongswan/plugins/wolfssl/wolfssl_plugin.c

```diff
@@ -24,7 +24,9 @@
 #include <library.h>
 #include <utils/debug.h>
 
+#define WOLFSSL_PLUGIN_INC_SSL_H
```

Comment:
I chose the define name `WOLFSSL_PLUGIN_INC_SSL_H` based on the prefix `WOLFSSL_PLUGIN` used in several of the files.  An alternative name, since this define controls logic in `wolfssl_common.h`, might be, `WOLFSSL_COMMON_INC_SSL_H`.  Please let me know if the alternative name is preferable.